packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -133,22 +135,31 @@ const AccountTransferForm = ({
     accounts_list.forEach((account, idx) => {
         const text = <AccountOption idx={idx} account={account} />;
         const value = account.value;
+        const currency = account.currency;
         (account.is_mt ? mt_accounts_from : accounts_from).push({
             text,
             value,
+            currency,
```

Comment:
why is this needed?

Reply:
We need to check "selected_from" currency with the `accounts_to.each(currency)`, but the value wasn't populated in the loop. So I've added it.